### PR TITLE
Stockmarket/carlos zamora2

### PR DESCRIPTION
**Pull request by CarlosZamora2181**
The table should now remain static throughout the website, let me know if there is an error but it should be good to go.

### CODE REVIEW

**Review by houyhourung — COMMENTED** [review]
please fix the merge conflicts as new changes are added

File: app.py [hunk 1]
```diff
@@ -73,12 +101,19 @@ def map_creator(latitude,longitude):
...
     stockData_url= "https://api.stockdata.org/v1/data/quote?"
     stockData= requests.get(stockData_url, params=parameters).json()
-    st.title(stockData["data"][0]["name"] + "'s Dashboard")
-    polystocks_url = "https://api.polygon.io/v3/reference/tickers/{}?apiKey=WJtsWZ032pndm6sfV4BAUnbaoOL7ku6X".format(
-        stock_ticker)
-    # Polygon.io Response from API
-    polyresponse = requests.get(polystocks_url).json()
-    information()
-    map()
+    if stockData["data"]:
```
Comment (houyhourung) [inline app.py]: i think you were pulling from my branch but it is not in the main branch so i would just keep the codes from the main branch as Alex just add codes in the main to fix the errors that I was trying to solve.

File: app.py [hunk 2]
```diff
@@ -11,20 +12,47 @@
...
+        ["Low", "High", "Market cap", "Currency", "Volume"])
+
+    # Line 96 is reading the CSV file with an updated file of the most popular stocks
+    popular_stocks = pd.read_csv('Popular_Stocks2.csv')
+    # Line 98 displays the data as a dataframe(interactive table)
+    st.dataframe(popular_stocks[["Company", "Ticker symbol", "Price"] + parameters_table])
+
+    st.caption("The information above can be displayed based on what the user wants to see.")
+
+    pass
```
Comment (houyhourung) [inline app.py]: please get rid of this line

File: app.py [hunk 3]
```diff
@@ -11,20 +12,47 @@
...
+
+
 #Input from the user in order to get a Stock
+
 userInput=  st.sidebar.text_input("Enter a valid stock ticker....GOOG")
+
 stock_ticker= userInput.upper()
 api_token= config.stockData_api_key
+interactive_table()
+
```
Comment (houyhourung) [inline app.py:35]: nitpicking but please get rid of the excessive spaces

File: app.py [hunk 4]
```diff
@@ -11,20 +12,47 @@
 #Streamlit SideBar Navigation
 st.sidebar.subheader("Stock Dashboard")
 
+
+def interactive_table():
```
Comment (houyhourung) [inline app.py:13]: this whole method should go before line 13 because when i am testing i see that this is blocking the checkboxes to be shown

**Review by houyhourung — CHANGES_REQUESTED** [review]

**Review by houyhourung — APPROVED** [review]

**Review by CarolinaMartine5 — APPROVED** [review]
I will go ahead and approve to merge into main. I have tested it and will be working to the overall viewing of the page.

**Pull request merged**